src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/FileUploadReviewViewModelToBulkUploadAddAndApproveDraftApprenticeshipsRequestMapper.cs

```diff
@@ -22,10 +22,13 @@ public async Task<BulkUploadAddAndApproveDraftApprenticeshipsRequest> Map(FileUp
         {
             var cacheModel = await _cacheService.GetFromCache<FileUploadCacheModel>(source.CacheRequestId.ToString());
             var csvRecords = cacheModel.CsvRecords;
+            var fileUploadLogId = cacheModel.FileUploadLogId;
+
```

Comment:
No test for this, but considering there are no tests for this class, I think we are allowed to ignore this